docs/realm.js

```diff
@@ -281,17 +281,37 @@ class Realm {
 
     /**
      * Initiate a write transaction.
+     *
+     * When doing a transaction, it is highly recommended to do error handling.
+     * If you don't handle errors, your data might become inconsistent. Error handling
+     * will often involve canceling the transaction.
+     *
+     * @example
+     * realm.beginTransaction();
+     * try {
+     *   realm.create('Person', { name: 'Arthur Dent',  origin: 'Earth' });
+     *   realm.create('Person', { name: 'Ford Prefect', origin: 'Betelgeuse Five' });
+     *   realm.commitTransaction();
+     * } catch (e) {
+     *   realm.cancelTransaction();
```

Comment:
```suggestion
     *   realm.cancelTransaction();
     *   throw e;
```
In a "real-life" scenario, you would probably either `console.error` or rethrow the exception.

Reply:
true